wagtail/contrib/redirects/templates/wagtailredirects/results.html

```diff
@@ -2,7 +2,7 @@
 {% if redirects %}
     {% if query_string %}
         <h2 role="alert">
-            {% blocktrans trimmed count counter=redirects.paginator.count %}
+            {% blocktrans trimmed count counter=redirects.count %}
```

Comment:
This seems to cause the pagination to not show the page count correctly:

<img width="94" alt="Page of . (missing the page numbers)" src="https://github.com/wagtail/wagtail/assets/6379424/7bec1384-5209-4e56-b8d5-384056006117">

Reply:
Yeah, I will inspect the data sent from the view again for the attribute to use instead of `count`. After refactoring `paginator.count ` doesn't work anymore as it is an object. It only works for the FBV